dockerfile_functions.sh

```diff
@@ -855,8 +856,8 @@ print_entrypoint() {
     cat "scripts/entrypoint.$2.sh" > "$dir/entrypoint.sh"
     chmod +x "$dir/entrypoint.sh"
     cat >> "$1" <<EOI
-COPY entrypoint.sh /
-ENTRYPOINT ["/entrypoint.sh"]
+COPY entrypoint.sh /__temurin_entrypoint.sh
+ENTRYPOINT ["/__temurin_entrypoint.sh"]
```

Comment:
```suggestion
ENTRYPOINT ["/__cacert_entrypoint.sh"]
```

Reply:
Is this really a good idea? For now, it's correct, but what if the entrypoint starts doing more?